src/Chainweb/Difficulty.hs

```diff
@@ -316,6 +316,8 @@ blockRate TestWithTime{} = Just $ BlockRate 4
 blockRate TestWithPow{} = Just $ BlockRate 10
 -- 120 blocks per hour, 2,880 per day, 20,160 per week, 1,048,320 per year.
 blockRate Testnet00 = Just $ BlockRate 30
+-- 120 blocks per hour, 2,880 per day, 20,160 per week, 1,048,320 per year.
+blockRate Testnet01 = Just $ BlockRate 30
```

Comment:
Hmmm, this pattern is making me wonder if we should bundle all these parameters up in a data structure so you can see all the params for a given testnet in one place.  Not for now though.  Maybe later.

Reply:
I understand the desire for that. The main thing opposing it is the usage patterns. We don't have an "application Monad", so there's no common `Env` running through the entire app. What most things have is a `ChainwebVersion` on hand (or they can easily find it via a nearby `Cut` or `BlockHeader`). Then with that, they can `O(1)` dispatch to any of these special values.

Reply:
Yeah, that makes sense.  Probably the right call.

Reply:
Putting all the possible configuration values (which sometimes are lambdas!) into one place would almost certainly violate some rule about separation-of-concerns.